src/main/java/org/folio/inventory/consortium/model/ConsortiumEnumStatus.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.inventory.consortium.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ConsortiumEnumStatus {
```

Comment:
I think this should be called "SharingStatus" or similar.

Also there might be overlap on this enum with another PR from @RomanChernetskyi

Reply:
Completed.